src/nvim/popupmenu.c

```diff
@@ -1007,15 +1015,18 @@ win_T *pum_set_info(int selected, char *info)
     }
     wp->w_topline = 1;
     wp->w_p_wfb = true;
+    wp->w_buffer->b_p_ts = curbuf->b_p_ts;
```

Comment:
```suggestion
```

I don't think it's worth only copying `&ts`; tab padding may also be affected by `&vts` (plus `&list` and `&lcs`).  
Also this code is only used by `nvim__complete_set`, so this change won't affect the info popup in the usual case. (`pum_set_selected`)

I think it's OK to not copy anything and just leave the buffer to use the global values.